docs/beaver-builder/about-this-release.md

```diff
@@ -2,54 +2,133 @@
+
+![Menu module centered inline logo, 2.5 features](/img/2-5-features-menu-module-centered-inline-logo.png) 
+
+#### Option to add search
+
+On the **General** tab, there's a new section to add search to the menu:
+
+![Menu module, add search, 2.5 features](/img/2-5-features-menu-module-search-option.png) 
+
+#### Support for WooCommerce Dynamic Cart 
+If you have WooCommerce installed, there's a section on the Menu module **General** tab to display the WooCommerce Dynamic Cart. You can display the number of items, the total amount, or both.
+
+This screenshot shows the difference between the **Cart** menu item added to the WordPress menu at **Appearance > Menus** versus the dynamic cart added with the Menu module:
+
+![Menu module WooCommerce dynamic cart, 2.5 features](/img/2-5-features-menu-module-woocommerce-cart.png) 
+
+<!---  ### Posts module
+
+The **No posts found** option has been moved to the **Content** tab.
+-->
+### Posts Slider module
+
+Now includes the standard Beaver Builder [Typography section](/beaver-builder/styles/typography/typography.md) for the **Post Title**, **Post Info**, and **Post Content** sections.
+
+<!--### Posts Slider & Posts Carousel modules
+
+There's now an option to customize the output when no posts are found.
+-->
+### Pricing Table module 
+
+The Pricing Table module has a number of enhancements, summarized in this screenshot and explained in the following sections.
+
+![Pricing Table module 2.5 features](/img/2-5-features-pricing-table-overview.png) 
+
+#### Display a ribbon
+You can select an option to display a ribbon with custom text in the upper left or right corner of a price column.
+
+#### Monthly/yearly pricing
+There is a new **Toggle pricing** option where you can choose  a single price or a toggle to display monthly and yearly pricing.
+
+#### Icons for each feature
+You can assign icons to individual features. Icons added to an individual feature override a general icon if one has been set for all features.
+
+#### Typography options
+
+Each feature in a Pricing Box now has its own **Style** tab with the standard Beaver Builder [Typography section](/beaver-builder/styles/typography/typography.md). 
+
+Typography sections have also been added to the **Title** and **Price** in each pricing box and the **Toggle Pricing** icon at the top of the table.
+
+### Video module
 
-## Beaver Builder Lite improvements
+There are two additional options on the **Structured data** tab for `contentUrl` and `embedUrl`.
 
-Three modules have been added to the free version of Beaver Builder:
+## Column improvements
 
-* [Heading](/beaver-builder/layouts/modules/heading.md)
-* [Icon](/beaver-builder/layouts/modules/icon-and-icon-group.md)
-* [Button](/beaver-builder/layouts/modules/button/button.md)
+* You can now set the width for single columns.  
+![Price Table module 2.5 features](/img/2-5-features-single-column-width.png) 
+<!-- * There are now custom positions available for backgrounds. -->
```

Comment:
```markdown
<!-- * There are now custom positions available for backgrounds. -->
```
Is this supposed to be commented out?

Reply:
Yes because I couldn't find it in the UI--if it's there will you tell me where it is?